osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-external-interface.xjb

```diff
@@ -0,0 +1,52 @@
+<jxb:bindings version="2.0" xmlns:jxb="http://java.sun.com/xml/ns/jaxb"
+    xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    jxb:extensionBindingPrefixes="xjc">
+
+   <jxb:bindings schemaLocation="common.xsd">
+      <jxb:schemaBindings>
+         <jxb:package name="org.opensmartgridplatform.adapter.ws.schema.smartmetering.common" />
+      </jxb:schemaBindings>
+   </jxb:bindings>
+   <jxb:bindings schemaLocation="sm-bundle.xsd">
+      <jxb:schemaBindings>
+         <jxb:package name="org.opensmartgridplatform.adapter.ws.schema.smartmetering.bundle" />
+      </jxb:schemaBindings>
+      <jxb:bindings node="/xs:schema/xs:complexType[@name='Actions']/xs:sequence/xs:choice">
+         <jxb:property name="ActionList" />
+      </jxb:bindings>
+      <jxb:bindings node="/xs:schema/xs:complexType[@name='AllResponses']/xs:sequence/xs:choice">
+         <jxb:property name="ResponseList" />
```

Comment:
Why do we need to rename Actions and AllResponses to ActionList and ResponseList? If it's not necessary I suggest either removing this customization or renaming the fields in the XSD's